uxarray/grid/grid.py

```diff
@@ -1890,6 +1893,21 @@ def get_spatial_hash(
 
         return self._spatialhash
 
+    def get_r_tree(self, reconstruct: bool = False):
+        """Build or retrieve a spatialpandas.spatialindex.HilbertRtree over per-face AABBs.
```

Comment:
Instead of using "per-face AABBs" in the docstring, can you mention that it is built on the bounding boxes?